src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    private static final Random random = new Random();
+    private final Color[] color = Color.values();
```

Comment:
Since `color` is a constant reference to the array of `Color` enum values, it should be named in all upper case to follow Java constant naming conventions, e.g., `COLORS`. Also, it could be static since its content is not tied to a particular instance of `ColorSupplier`.